tests/distributions/test_continuous.py

```diff
@@ -2369,6 +2369,25 @@ def seeded_weibul_rng_fn(self):
         "check_rv_size",
     ]
 
+    # See issue #7220
```

Comment:
Nitpick, comment inside the test function is less likely to be lost/out of place in the future